tgstation.dme

```diff
@@ -745,6 +745,7 @@
 #include "code\game\objects\items\cosmetics.dm"
 #include "code\game\objects\items\courtroom.dm"
 #include "code\game\objects\items\crayons.dm"
+#include "code\game\objects\items\dakis.dm"

```

Comment:
code/game/objects/items/storage/dakis.dm???

Reply:
they store items, what's wrong?

Reply:
>not hiding bear traps in your waifu to protect her against brainless orc assistants

??

Reply:
![wea8ol7](https://user-images.githubusercontent.com/6209658/31264487-7b5f92fe-aa37-11e7-93e6-86e19eaad166.png)
![8wbndtc](https://user-images.githubusercontent.com/6209658/31264491-81b2a678-aa37-11e7-90cb-762cbfbea39c.png)
![zzcjoyo](https://user-images.githubusercontent.com/6209658/31264494-907fe936-aa37-11e7-8946-4b168e82b919.png)

Reply:
shiiiiii how do i fix this one second

Reply:
FIXED